### PR TITLE
Downgraded spf13/viper

### PR DESCRIPTION
**Pull request by skhalash**
Downgraded spf13/viper to the version used by the cli to avoid dependency problems. As soon as we have our changes ready, we will bump the versions of spf13/cobra and spf13/viper in the cli as well as the reconciler.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,22 +13,30 @@ require (
 	github.com/iancoleman/strcase v0.1.3
 	github.com/imdario/mergo v0.3.12
 	github.com/jonboulle/clockwork v0.1.0
+	github.com/json-iterator/go v1.1.11 // indirect
```
Comment (tobiscr) [inline go.mod:16]: Can't this be done by using `replace` in the CLI go.mod ?

Comment (tobiscr) [reply go.mod:16]: https://golang.org/ref/mod#go-mod-file-replace

**Review by tobiscr — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (tobiscr) [reply go.mod:16]: This comment is for all downgrades intended .. don'T want to repeat it several times :)

**Review by tobiscr — CHANGES_REQUESTED** [review]

**Comment by skhalash** [comment]
Good idea, @tobiscr. I will replace the dependency on the cli side.

**Pull request closed without merging**